### PR TITLE
chore(ci): retrieve APK url in github workflow

### PR DESCRIPTION
**Pull request by matejkriz**
## Description

- Get APK url in github workflow so it can be uploaded anywhere
- Delete hook to upload APK on github

Example run: https://github.com/trezor/trezor-suite/actions/runs/12312464741/job/34364557163#step:6:38 (with preview build because of permissions)

## Related Issue

Related [#5728](https://github.com/trezor/trezor-suite/issues/5728)

### CODE REVIEW

File: suite-native/app/eas-post-success.sh [hunk 1]
```diff
@@ -16,60 +16,8 @@ distribute_develop_apk() {
...
-
-if [[ "$EAS_BUILD_PLATFORM" == "android" ]]; then
-    if [[ "$EAS_BUILD_PROFILE" == "develop" ]]; then
-        distribute_develop_apk
-    elif [[ "$EAS_BUILD_PROFILE" == "productionAPK" ]]; then
-        create_release_draft
-        upload_production_apk
-    fi
+if [[ "$EAS_BUILD_PLATFORM" == "android" && "$EAS_BUILD_PROFILE" == "develop" ]]; then
+    distribute_develop_apk
```
Comment (vytick) [inline suite-native/app/eas-post-success.sh:20]: Do we want develop here?

Comment (matejkriz) [reply suite-native/app/eas-post-success.sh:20]: This uploads develop builds on firebase so we can download it via App Tester.

Comment (vytick) [reply suite-native/app/eas-post-success.sh:20]: Roger 🫡

**Review by vytick — APPROVED** [review]
lgtm, great stuff! 🚀

**Comment by vdovhanych** [comment]
> lgtm, great stuff! 🚀

This is not done yet :D I will add an app upload to the release site data.trezor.io so it can be downloaded. Or we can do it in a different PR i don't care.

**Comment by matejkriz** [comment]
This PR so far does nothing, it only removes the option to create a github release for mobile app and prepares the data to be published elswhere...

**Comment by matejkriz** [comment]
followup here https://github.com/trezor/trezor-suite/pull/16008

**Pull request merged**